modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -269,6 +269,14 @@ ifdef::ibm-z,ibm-z-kvm[IBM Z infrastructure.]
 ifdef::ibm-power[IBM Power Systems infrastructure.]
 ifdef::rhv[RHV infrastructure.]
 ifndef::openshift-origin[]
++
+[WARNING]
+====
+Red Hat Virtualization does not currently support installation with user provisioned infrastructure to the ovirt platform. Therefore, you must set the platform to `none`, allowing OpenShift to identify each node as a bare-metal node and the cluster as a bare-metal cluster. This is the same as xref:../../installing/installing_platform_agnostic/installing-platform-agnostic.adoc#installing-platform-agnostic[installing a cluster on any platform], and has the following limitations:
```

Comment:
Please use "user-provisioned infrastructure".

I believe "ovirt" should be oVirt to match other instances of that term in general text. (In inline-code instances, it seems overwhelmingly to be all lowercase.)

Are you sure that you want "to the oVirt platform" rather than "on the oVirt platform"?

"OpenShift" alone is usually replaced.  In most cases, you can replace with `{product-title}`.  I don't know enough about the topic to know if that is the best replacement or if the software is the installer.